theme/static/css/hydroshare_core.css

```diff
@@ -4650,4 +4650,16 @@ div#resource-content {
 
 .btn-choose-coordinates {
 	vertical-align: top;
+}
+
+/* popover styling */
+.popover-content {
+  padding: 9px 14px;
+  font-size: 14px;
+}
```

Comment:
Add `line-height: 1.5;` for better readability.

Reply:
agreed. Will update the change.